packages/db/src/system.ts

```diff
@@ -47,10 +47,12 @@ export type ConnectOptions = Meta.ConnectOptions<Collections>;
 
 const system = Meta.forDefinitions(definitions);
 
-export const connect: (options: TruffleConfig | ConnectOptions) => Db =
-  system.connect;
+export const connect: (
+  options: TruffleConfig | ConnectOptions | undefined
+) => Db = system.connect;
 
-export const serve: (options: TruffleConfig | ConnectOptions) => ApolloServer =
-  system.serve;
+export const serve: (
+  options: TruffleConfig | ConnectOptions | undefined
```

Comment:
(Another suggestion for batch :)

```suggestion
  options?: TruffleConfig | ConnectOptions
```